trollmoves/utils.py

```diff
@@ -19,10 +19,16 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Utility functions for Trollmoves."""
+
 import socket
 from urllib.parse import urlparse, urlunparse
 
-import netifaces
+try:
+    import netifaces
+except ImportError:
+    netifaces = None
```

Comment:
I really don't like this, sorry. Could we just move `get_local_ips` to client.py instead, where we know netifaces are a requirement?

Reply:
Fixed in https://github.com/pytroll/trollmoves/pull/129/commits/096231c40ac185207d7fa3827bc9b881ce6c41a5